src/Modules/OrchardCore.Commerce.Tax/Views/TaxRateSettings_Edit.cshtml

```diff
@@ -104,7 +66,7 @@
             </button>
```

Comment:
I don't want to be nit-picky, but I think the UI could be improved (sorry for not mentioning before):

1. I think these two buttons should be swapped. It seems more logical to me, to have the "up" button upper.
![image](https://user-images.githubusercontent.com/80963259/212335415-262b1d5d-9828-4f34-8071-1c58659b7bd0.png)

2. On my screen the rows are too long. I don't know if we can fix it without making the UI look stupid or cluttered:
![image](https://user-images.githubusercontent.com/80963259/212335648-89190401-e407-4429-8711-cc2f8464d004.png)

Reply:
I see. Alright, thank you!

Reply:
I made the two Address columns hidden by default, and added a checkbox to reveal them if needed. They are most likely to not be used (I didn't want to over-complicate this with more granularity and the rest is still likely relevant - city, state or posttal code may be used for regional taxes). This way it should fit on most desktop screens without scrolling. The user can always zoom out if they have a low resolution display.

![image](https://user-images.githubusercontent.com/4007293/212441693-042c1739-c4ad-4e32-aab7-38e9ff643c39.png)